### PR TITLE
Generic Longitudinal updates

### PR DESCRIPTION
**Pull request by fbertsch**
Basically added three new options to the CLI, as well as increasing test coverage to include options. These options put this code in line with the GenericCountView PR.

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/GenericLongitudinal.scala [hunk 1]
```diff
@@ -26,6 +30,15 @@ object GenericLongitudinalView {
...
        descr = "To submission date",
        required = true)
+    val submissionDateCol = opt[String](
+      "submission-date-col",
+      descr = "Name of the submission date column. Defaults to submission_date_s3",
+      required = false)
+    val selection = opt[String](
+      "select",
+      descr = "Select statement to retrieve data with; e.g. \"substr(subsession_start_date, 0, 10) as activity_date\". " +
+                "If none given, defaults to all input columns",
```
Comment (harterrt) [inline src/main/scala/com/mozilla/telemetry/views/GenericLongitudinal.scala]: Indentation looks off

Comment (fbertsch) [inline src/main/scala/com/mozilla/telemetry/views/GenericLongitudinal.scala]: I wasn't sure what to do about this so I just decreased the indent by one. Any prior art here?

Comment (harterrt) [inline src/main/scala/com/mozilla/telemetry/views/GenericLongitudinal.scala]: I'm happy with the revision as-is :). The [style guide](http://docs.scala-lang.org/style/indentation.html) suggests two spaces total.

**Comment by fbertsch** [comment]
@harterrt if you're happy with the changes I need you to r+ again :)

**Pull request merged**